.ci/docker/node-puppeteer/Dockerfile

```
@@ -1,5 +1,6 @@
 #Dockerfile for docker.elastic.co/observability-ci/node-puppeteer:12
-FROM node:12
+ARG NODEJS_VERSION
```

```suggestion
ARG NODEJS_VERSION=12
```

Perhaps it would be a good idea just to preserve the default here?